CMakeLists.txt

```diff
@@ -8,7 +8,9 @@ message(STATUS "Prefix Path: ${CMAKE_PREFIX_PATH}")
 
 list(APPEND CMAKE_MODULE_PATH ${CMAKE_CURRENT_SOURCE_DIR}/cmake)
 
-project(fineftp)
+project(fineftp VERSION 1.0.11)
```

Comment:
Hi Florian. I see that most of your comments are about the "aggressive" changes I mentioned in the description ... I knew that this would be controversial 🤓 .

I noticed that you had 2 separated folders with independent CMake code in your repository. One for the library and other one for an example of a consumer of fineftp. 
As I see it, normally those 2 "projects" would live in individual repositories, and the consumer project would not include the whole source code of the fineftp library but try to find an installation of it in the system via `find_package`. In that case, the consumer project would find the **export** file without problem in the include folder of fineftp.

Due to this unusual situation I saw in your repository, it was not trivial to find the **export** header from the consumer project. The reason is that each of the 2 projects have their independent subfolder in the build directory and therefore, I did not see how to tell to the consumer directory how to find the **export** file which is generated in the fineftp build directory.

I'll give it another try and check how I could make the integration without touching too much your original CMake code. However, if I do not find an elegant and clean solution I would propose you separate the **fineftp-server** and the **example** subfolder in separate repositories. 

I'll come back to you in few days 😉